main.go

```diff
@@ -28,6 +29,9 @@ func main() {
 		context.Rest.SetRoutes(c.Routes())
 	}
 
+	// Store self information in the store
+	peer.AddSelfDetails()
```

Comment:
AddSelfDetails is no more meaningful because Self can be volume, or might be other information. for making it more clear you can use AddSelf(peer/node)details()